backend/nodejs/src/services/MonitoringService.ts

```
@@ -0,0 +1,132 @@
+    }
+  }
+
+  public stopWakeCron(jobId: string) {
+    const job = this.wakeJobs.get(jobId);
+    if (job) {
+      job.stop();
+      this.wakeJobs.delete(jobId);
+    }
+  }
+
+  public async updateDeviceMonitoring(device: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)